src/Components/Home.jsx

```diff
@@ -1,9 +1,13 @@
-import React, { Component } from "react";
```

Comment:
what about component imported here?

Reply:
same here!

Reply:
the contents of the red are the ones that have been changed unnecessarily not added